### ISSUE TITLE
Chest pain reproducible on palpation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10134]
**Preferred term label:**
Chest pain reproducible on palpation

UMLS has a bunch of region-specific terms such as https://uts.nlm.nih.gov/uts/umls/concept/C3836911 but none for chest pain, which is what I want.

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Costochondritis versus myocardial infarction

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000047
- label: Chest pain triggered by palpitation
- parents: [Triggered by (HP:0025204)](https://hpo.jax.org/app/browse/term/HP:0025204)
- pmid: 
- def: Chest pain can be induced by palpation of the chest, which may be an indication of costochondritis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10134
- synonyms: Chest pain reproducible on palpation(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
Fixed, thanks!!
